app/ante/evm/ante_test.go

```diff
@@ -1148,6 +1148,7 @@ func (suite *AnteTestSuite) TestAnteHandlerWithParams() {
 		GasLimit:  100000,
 		GasFeeCap: big.NewInt(ethparams.InitialBaseFee + 1),
 		GasTipCap: big.NewInt(1),
+		Input:     []byte("create bytes"),
```

Comment:
are these used? I'd rather generate a more realistic payload

Reply:
For the purpose of this function thats being tested, it only checks if the input is not nil - thats why I didnt want to over complicate it